tests/foreman/cli/test_bootstrap_script.py

```diff
@@ -0,0 +1,148 @@
+# -*- encoding: utf-8 -*-
+"""Test for bootstrap script"""
+from robottelo.decorators import stubbed
+from robottelo.test import CLITestCase
+
+
+class BootstrapScriptTestCase(CLITestCase):
+    """Test class for bootstrap scipt."""
+
+    @stubbed()
+    @classmethod
+    def setUpClass(cls):
+        """ create VM for testing """
+
+    @stubbed()
+    def test_positive_register(self):
+        """@test: system is registered
+
+        @Feature: Bootstrap Script
+
+        @Setup: none
```

Comment:
Setup section is not mandatory, you can drop the section.  This comment applies to the entire module.